src/client/canvas/svg/arc.tsx

```diff
@@ -0,0 +1,67 @@
+import { observer } from 'mobx-react'
+import * as React from 'react'
+import { Component } from 'react'
+
+import { ArcGeometry } from '../geometry/arc_geometry'
+import { Shape } from '../object/shape'
+
+import { appearance } from './svg'
+
+@observer
+export class Arc extends Component<{model: Shape<ArcGeometry>}> {
+  // TODO draw arcs
+  public render() {
+    const m = this.props.model
+    const g = m.geometry
+    // g.anticlockwise
+    // g.startAngle
+    // g.endAngle
+    // g.origin
+    // g.radius
+    // rx ry x-axis-rotation large-arc-flag sweep-flag x y
+    return <path d={this.arc(g.origin,
+                             g.radius,
+                             g.startAngle,
+                             g.endAngle,
+                             g.anticlockwise)} {...appearance(m.appearance)} />
+  }
+
+
+  private arc(origin: {x: number, y: number},
+              r: number,
+              a0: number,
+              a1: number,
+              ccw: boolean): string {
+
+    // Is this arc empty?
+    if (r === 0) {
+      return ''
```

Comment:
Should probably move this into `render()` so that a path isn't even rendered.

e.g. `render() { return r !== 0 && <path d={...}/> }`